### DISCUSSION TITLE
how to cancel MOC order?

### DISCUSSION THREAD

**Discussion by rgib37190** [ib-api-reloaded/ib_async#79] [Q&A]
I use the following code to place my order, but I didn't know how to cancel it. I use openOrders method but got empty.
And my order status is PendingSubmit. 
`Order(action=action, orderType="MOC", totalQuantity=quantity, algoStrategy='Adaptive')`

**Comment by cheukhin1024** [comment]
Use `cancelOrder()` function. Check the ib_async doc.
![image](https://github.com/user-attachments/assets/8c471948-2e6b-44ff-9908-f6ed87ad439f)
